### PR TITLE
add dev env for githubscript

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: githubscript/docker.d/worker.yml [hunk 1]
```diff
@@ -20,4 +20,10 @@ github_projects:
         github_token: dummy
         github_owner: non-existing-owner
         github_repo_name: non-existing-repo
+    'COT_PRODUCT == "mobile" && ENV == "dev"':
+      mock:
+        allowed_actions: ["release"]
+        github_token: dummy
```
Comment (JohanLorenzo) [inline githubscript/docker.d/worker.yml:31]: We could use `{ "$eval": "GITHUB_TOKEN_WRITE_ACCESS_STAGING" }` here. This way, we actually exercise code that uploads something onto a github repo. What do you think?

File: githubscript/docker.d/worker.yml [hunk 2]
```diff
@@ -20,4 +20,10 @@ github_projects:
...
+        github_owner: non-existing-owner
```
Comment (JohanLorenzo) [inline githubscript/docker.d/worker.yml:32]: Same here, we could wire this to your fork, so we sure this works end-to-end. The `mock` product is quite simple doesn't do much.

**Review by JohanLorenzo — APPROVED** [review]
This looks good to me. We can use the `mock` product against the dev workers. Although I think we could use this type of instance to actually write on a github thanks to https://github.com/mozilla-release-automation-bot-staging. 

I'm fine with both options, so feel free to choose the one you prefer.

**Comment by escapewindow** [comment]
Added the `android-components` repo, pointing at my fork, for both staging and dev. Depends on https://github.com/mozilla-services/cloudops-infra/pull/2752 .

**Pull request merged**